tests/jenkins/test_validation.py

```diff
@@ -45,13 +49,14 @@ def create_output_files(samples: Path, root: Path, sequencing_technology: str):
 )
 def test_compare_merged_output_file(
     test_data_path,
+    pathogen,
     results_csv,
     expected_results_csv,
     exc,
 ):
-    actual_path = test_data_path / "jenkins" / "validation" / results_csv
-    expected_path = test_data_path / "jenkins" / "validation" / expected_results_csv
-    data = data_config["sars_cov_2"]["config"]
+    actual_path = test_data_path / "integration_test_validation" / pathogen / results_csv
+    expected_path = test_data_path / "integration_test_validation" / pathogen / expected_results_csv
```

Comment:
If the start of this path is identical, is it worth wrapping it in a fixture to make it easier to update in the future?

Reply:
Will do as a separate commit.

Reply:
See commit: `PSG-2361: use fixtures for common test data paths`